plutus-contract/test/Spec/golden/traceOutput - pubKeyTransactions.txt

```diff
@@ -90,39 +90,37 @@
                          80a4f45b56b88d1139da23bc4c3c75ec6d32943c087f250b86193ca7
                        Utxo index:
 [INFO] Slot 2: W[2]: Finished balancing:
-                       Tx 41d625f32bf13e52ca6d42896ffb31e97883259fd15db239ffe86d2371e3ad67:
+                       Tx 414d8f64e6635c954140ef931593457551fd9ede478ea420327afe8e72e0d7fd:
                          {inputs:
-                            - 38fbe171e83c33581a737d0d1efa064afe389c8d49049e609a8ea311738a834e!0
-
                             - 43ba666cc8a22a04b63a3b605ce14146dfa5ed999986625ad90c1bc16dabdd84!20
 
                          reference inputs:
                          collateral inputs:
                          outputs:
                            - 5000000 lovelace addressed to
                              PubKeyCredential: 2e0ad60c3207248cecd47dbde3d752e0aad141d6b8f81ac2c6eca27c (no staking credential)
-                           - 9822575 lovelace addressed to
+                           - 4828603 lovelace addressed to
```

Comment:
How did this value change?

Reply:
Fewer inputs, smaller transaction, lower fees!